husky_directory/blueprints/search.py

```diff
@@ -14,6 +15,18 @@ def __init__(self, logger: Logger):
         super().__init__("search", __name__, url_prefix="/search")
         self.logger = logger
         self.add_url_rule("/", view_func=self.search)
+        self.add_url_rule("/render", view_func=self.render_search)
+
+    def render_search(self, request: Request, search_service: DirectorySearchService):
```

Comment:
Line 20: I struggled with making the existing search function render and send data and still have the existing test pass. 
I created a new test to cover the function I added.

Reply:
Sorry, but I don't think this change is appropriate. The search function below should be doing this. If not, that needs to be fixed.

Reply:
Feel free to let me know _what_ wasn't working.

Reply:
Thanks for the feedback. Without this change, when the form submit returns, the user sees just the raw data object. The intention of my change was to return the data (with a small tweak) and render index.html where the data is parsed along with the rest of the page.

Reply:
OK. Yes, the intent of the `search` api was to function as an API for the front-end to call and render the JSON result. 
If instead the front-end is just posting a form, the intent of your change makes sense, but I don't think a copy-paste with one line difference is the right solution.

Are you able to just call the search method to get its result, then render it.

```
def render(self, request: Request, service: DirectorySearchService):
        return render_template("index.html", search_result=self.search(request, service).get_json())
```

Reply:
> I don't think a copy-paste with one line difference is the right solution.
Yeah, didn't think so, I tried one approach without introducing a new function, didn't work for a couple of reason. What you propose here looks like it should work, I'll plug it in.

Reply:
if it doesn't work, let me know.

Reply:
this is great, creating PR

Reply:
awesome!